docs/sdk/dev/development-tools/code-sniffer.md

```diff
@@ -51,7 +51,7 @@ related:
     link: docs/scos/dev/sdk/development-tools/tooling-config-file.html
 ---
 
-To correspond to [PSR-2 standards](http://www.php-fig.org/psr/psr-2/), we integrated the well known [PHP-CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) and [PHPCodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer).
+To correspond to [PSR-2+ standards](http://www.php-fig.org/psr/psr-2/), we integrated the well known [PHPCodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer).
```

Comment:
PSR-2+ can be confusing. All other PSRs higher than 2, except of 12, have no relation to code styling. 

So maybe better to write "[PSR-2 standards](http://www.php-fig.org/psr/psr-2/) with additions" or something like that.